### DISCUSSION TITLE
MacOS Server: Auto start server & auto hide start screen

### DISCUSSION THREAD

**Discussion by burcaka** [input-leap/input-leap#1660] [Q&A]
### What happened?

I've built the input-leap binary from source files and it works like a charm. Except...

When you start input-leap, the "start screen" is displayed. You have to press the start button manually and close it to the menu bar, again, manually.

![Screen Shot 2023-06-08 at 11 11 46](https://github.com/input-leap/input-leap/assets/33871933/61675cc1-ccaf-44c3-bd62-29225feab701)

Default behaviour with Barrier was:
(1) User logs in to Mac.
(2) Barrier automatically starts last (default) server.
(3) No "start screen" is shown and barrier is minimized to the menu bar.

Is there a way to replicate this behaviour by using:
(1) Option(s) during build,
(2) via the conf file or,
(3) running a script?

Thanks for the great work.

### Version

v2.4.0

### What OSes are you seeing the problem on? (Check all that apply)

macOS

### What OS versions are you using?

MacOS Catalina 10.15.7

**Comment by shymega** [comment]
I would say this is a regression from refactoring. Kindly create an issue with logs and other details, in the meantime, I'll close and lock this discussion.
